### ISSUE TITLE
Binary builds from 1.6.1 on do not seem to run on OSX 10.7.5 

### ISSUE THREAD

**Issue by jbrasla** [fikovnik/ShiftIt#168]
When trying to run the binary get the message "You have Mac OS X 10.7.5. The application requires Mac OS X 10.8 or later."  Is 1.6 the latest that runs on Mac OS X 10.7.5?  Thanks.

**Comment by fikovnik** [comment]
It is although it should still be possible to compile 1.6.3 on 10.7. If you have Xcode you can give it a try. Basically you only need to change the base SDK in the ShiftIt target.

**Comment by xenyz** [comment]
I tried on OS X 10.7.5, XCode 4.6.3 and ran into an error with PreferencesWindow.xib. It asks me to use a newer version of XCode, but my old Macbook Pro can only run Lion.

Here's the build error, but I doubt you need it:

CompileXIB Base.lproj/PreferencesWindow.xib
    cd /Users/adam/Downloads/ShiftIt-version-1.6.3/ShiftIt
    setenv XCODE_DEVELOPER_USR_PATH /Applications/Xcode.app/Contents/Developer/usr/bin/..
    /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --errors --warnings --notices --plugin ShortcutRecorder.framework/Versions/A/Resources/ShortcutRecorder.ibplugin --output-format human-readable-text --compile /Users/adam/Library/Developer/Xcode/DerivedData/ShiftIt-etcpcuvdysyhdlaqpzbkawuxwdce/Build/Products/Release/ShiftIt.app/Contents/Resources/Base.lproj/PreferencesWindow.nib /Users/adam/Downloads/ShiftIt-version-1.6.3/ShiftIt/Base.lproj/PreferencesWindow.xib --sdk ""

<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"><plist version="1.0"><dict>	<key>com.apple.ibtool.errors</key>	<array>		<dict>			<key>description</key>			<string>Illegal invocation. Try 'man ibtool' for more information.</string>		</dict>	</array></dict></plist>Command /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool failed with exit code 1

**Comment by iEFdev** [comment]
I'd also like to see support from 10.6+ on the newer versions.
// Not using xcode anymore, just have the clitools.

**Comment by jbrasla** [comment]
Thanks for the follow-ups.  I tried compiling with Xcode (Version 4.6.3) for OS X 10.7.5, which failed with the following output (similar to xenyz's):

CompileXIB Base.lproj/MainMenu.xib
    cd /Users/myuser/Applications/ShiftIt/ShiftIt
    setenv XCODE_DEVELOPER_USR_PATH /Applications/Xcode.app/Contents/Developer/usr/bin/..
    /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --errors --warnings --notices --plugin ShortcutRecorder.framework/Versions/A/Resources/ShortcutRecorder.ibplugin --output-format human-readable-text --compile /Users/myuser/Library/Developer/Xcode/DerivedData/ShiftIt-ablpmrsjbhbgrpbulowrjhrpnnlo/Build/Products/Release/ShiftIt.app/Contents/Resources/Base.lproj/MainMenu.nib /Users/myuser/Applications/ShiftIt/ShiftIt/Base.lproj/MainMenu.xib --sdk ""

<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd"><plist version="1.0"><dict>	<key>com.apple.ibtool.errors</key><array><dict><key>description</key>	<string>Illegal invocation. Try 'man ibtool' for more information.</string></dict></array></dict></plist>Command /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool failed with exit code 1

**Comment by fikovnik** [comment]
Well, I guess, the easiest is to manually install the version that still work. It should be possible for me to get 10.6 / 10.7 SDK from here http://devernay.free.fr/hacks/xcodelegacy/ but right now I'm busy with paper deadlines and won't be able to work on it.

**Comment by Brainzyy** [comment]
Any news on this?

**Comment by fikovnik** [comment]
No, not really, I do not have access to any older OS X version so I cannot progress on it. Any patches are of course welcomed.

**Comment by Brainzyy** [comment]
The reason we can't compile @jbrasla @xenyz is because the xib files are defined for xcode5 which we cannot use. @fikovnik you can change this very easily in xcode, could you send over the xib files that can be opened in xcode 4?

**Comment by fikovnik** [comment]
Could you a bit more specific how can I export the xibs into Xcode 4?

**Comment by Brainzyy** [comment]
@fikovnik 1. Right-Click the xibs
2. Open file inspector
3. On Interface Builder Documents section change Open with to Xcode 4.6.
3. Save

**Comment by fikovnik** [comment]
Thanks for the howto. Unfortunately, the earliest Xcode is 5.1. Will that be any helpful?

**Comment by Brainzyy** [comment]
Should be perfect.

**Comment by fikovnik** [comment]
OK, pushed in develop branch 76e6743.

**Comment by Brainzyy** [comment]
@fikovnik I made some progress. I was able to compile everything with no problems on 10.8.5 and it works on there. But when I compile it for 10.7 and then try to run my compiled version, the console shows these errors: 
ShiftIt: Unable to load nib file: MainMenu, exiting
com.apple.launchd.peruser.501: ([0x0-0x22022].org.shiftitapp.ShiftIt[1433]) Exited with code: 1
I'll try to export the project for 10.7 and then try to compile it on 10.7 as well and report back.
